### PR TITLE
test: added integration tests for Django

### PR DESCRIPTION
**Pull request by gauravsnj**

### CODE REVIEW

File: samples/python/django/sample.py [hunk 1]
```diff
@@ -145,6 +146,8 @@ def transaction_rollback():
 if __name__ == "__main__":
 
   try:
+    tables_created = 0
```
Comment (olavloite) [inline samples/python/django/sample.py]: nit: could this be a boolean?

File: src/test/java/com/google/cloud/spanner/pgadapter/python/django/ITDjangoTest.java [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    }
+    testEnv.stopPGAdapterServer();
+    testEnv.cleanUp();
+  }
+
+  @Test
+  public void test() throws IOException, InterruptedException {
+    String actualOutput = executeIntegrationTests();
+    String expectedOutput = "Django Test Completed Successfully\n";
+    assertTrue(actualOutput.contains(expectedOutput));
```
Comment (olavloite) [inline src/test/java/com/google/cloud/spanner/pgadapter/python/django/ITDjangoTest.java]: nit: It's good to add the `actualOutput` as the first argument to a call to `assertTrue`. That will cause JUnit to print out the contents of `actualOutput` if the assertion fails, which can make it easier to determine the reason that the assertion failed (especially if the assertion fails during a CI run).

File: src/test/java/com/google/cloud/spanner/pgadapter/python/django/ITDjangoTest.java [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+      writer.write(originalDjangoSetting);
+    }
+    testEnv.stopPGAdapterServer();
+    testEnv.cleanUp();
+  }
+
+  @Test
+  public void test() throws IOException, InterruptedException {
+    String actualOutput = executeIntegrationTests();
+    String expectedOutput = "Django Test Completed Successfully\n";
```
Comment (olavloite) [inline src/test/java/com/google/cloud/spanner/pgadapter/python/django/ITDjangoTest.java]: nit: Could we change the text at the end of the sample to
```suggestion
    String expectedOutput = "Django Sample Completed Successfully\n";
```

**Review by olavloite — APPROVED** [review]

File: samples/python/django/sample.py (hunk 1 above)
Comment (gauravsnj) [reply samples/python/django/sample.py]: Done

File: src/test/java/com/google/cloud/spanner/pgadapter/python/django/ITDjangoTest.java (hunk 2 above)
Comment (gauravsnj) [reply src/test/java/com/google/cloud/spanner/pgadapter/python/django/ITDjangoTest.java]: Done

File: src/test/java/com/google/cloud/spanner/pgadapter/python/django/ITDjangoTest.java (hunk 3 above)
Comment (gauravsnj) [reply src/test/java/com/google/cloud/spanner/pgadapter/python/django/ITDjangoTest.java]: Done

**Pull request merged**